Packs/ShiftManagement/Scripts/GetRolesPerShift/GetRolesPerShift.py

```diff
@@ -0,0 +1,73 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+from typing import List
+
+HOURS_DAYS_HEADER = 'Hours / Days'
+SUNDAY_HEADER = 'Sunday'
```

Comment:
Wonder if there is a way to get them from i18n - probably not...

Reply:
nope